src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt

```diff
@@ -0,0 +1,44 @@
+package org.vaccineimpact.orderlyweb.customConfigTests
```

Comment:
One thing that isn't clear to me - the default scope of the token requested by the built in GitHubClient is "user" - I thought we needed more scopes than that to access team membership? At least judging from what I've written here: https://github.com/vimc/orderly-web/blob/master/src/app/src/test/resources/spec/spec.md#post-login 

This is the sort of thing that will only get picked up by an end to end test, although maybe there is a test missing from `GithubApiClientAuthHelper` about required token scopes. I see we have a test called `" CredentialsException is thrown if email is not public and token does not have email scope"`, but we don't have ` "CredentialsException is thrown if team membership is required and token does not have admin:org scope"`.

Reply:
Looks like change made for mrc-203 should have fixed this so that we now only need user scope for org check (and we should have been ok for team anyway). Will add test for read:user scope.